src/Service/Resolvers/BaseSqlQueryBuilder.cs

```diff
@@ -368,7 +373,7 @@ public string JoinPredicateStrings(params string?[] predicateStrings)
 
             if (!validPredicates.Any())
             {
-                return "1 = 1";
+                return BASE_PREDICATE;
```

Comment:
can you add comment for parameter `params string?[] predicateStrings` in summary?